app/views/propertyLinking/relationshipToProperty.scala.html

```diff
@@ -90,21 +94,16 @@ <h1 class="govuk-heading-l">@messages("declarePropertyRelationship.title"):</h1>
                     classes = "govuk-fieldset__legend--m"
                 ))
             )),
-            idPrefix = Some("capacity"),
             name = "capacity",
             items = CapacityType.options.map { item =>
                 RadioItem(
                     value = Some(item),
-                    content = Text(messages(s"capacity.$item")),
-                    checked = form.data.get("capacity").fold(false)(_ == item)
+                    content = Text(messages(s"capacity.$item"))
                 )
             },
-            errorMessage =  getFiledErrorMessage("capacity"))
```

Comment:
errorMessage =  getFiledErrorMessage("capacity") might not required as you have added .withFormField(form("capacity"))

Reply:
done - and removed other redendant code

Reply:
thanks mate